docs/deep-dives/specifications/fml-spec.mdx

```diff
@@ -481,17 +501,54 @@ features:
                 description: A pre-bundled image
                 type: Image
                 default: ic_fox
-                suggested:
-                  ic_fox: A picture of a cute fox
-                  ic_snow_fox: A picture of a cute snow fox
-                  ic_redpanda: A picture of a cute red panda
             message-content:
                 description: Text content to show the user
                 default: msg_thankyou
-                suggested:
-                  msg_welldone: A nice well done for upgrading
-                  msg_thankyou: A nice thank you for sticking with us
-                  msg_warning: A warning that bad things happen to good people
+```
+
+<Tabs
+  defaultValue="swift"
+  values={[
+    { label: "Swift", value: "swift" },
+    { label: "Kotlin", value: "kotlin" },
+  ]
+}>
+<TabItem value="swift">
+Values for `Text` are strings which must correspond to the following format:
+
+`KEY` |
+`TABLE_NAME` '`/`' `KEY`
+
+The lookup uses the `bundle.localizedString(forKey: key, value: nil, table: tableName)` method.
+
+If the text is not found, then the raw string is used instead.
+
+For `Image`, they should correspond to an image named in an asset bundle, and use the `UIImage(named: name, in: bundle, compatibleWith: nil)` constructor.
+
+</TabItem>
+<TabItem value="kotlin">
+
+Values for `Text` and `Image` are strings which must correspond to the following format:
+
+`[a-z][a-z_0-9]*`
+
+The lookup uses the `R` class.
+
+</TabItem>
+</Tabs>
```

Comment:
It's kinda difficult to tell where this tab block ends when rendered, could it be worth it to add a horizontal rule or something else to more clearly separate it?

https://user-images.githubusercontent.com/4909307/199591921-b2b40d81-1727-4ad7-9de1-f591a4cc3205.mov

Reply:
Yes!